showtime-xyz/showtime-frontend
# feat: EAS update & add PanToClose component

# Why

- Currently, the EAS update is not friendly enough.
- Snackbar/Toast components don't support gestures to close.

# How

- Record the time when the App switches to the background via `mmkv` and optimize show Snackbar update timing.
- add a `PinchToClose` component and wrap the Snackbar and Toast, but now only support Native.


# Test Plan
- check if the EAS update logic works.
- check if Snackbar/Toast swipe to close works.
